contracts/debt/TokemakIncurDebt.sol

```diff
@@ -0,0 +1,305 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.4;
+
+import "../interfaces/IOHM.sol";
+import "../interfaces/IsOHM.sol";
+import "../interfaces/IgOHM.sol";
+import "../interfaces/IERC20.sol";
+import "../interfaces/IStaking.sol";
+import "./interfaces/ITreasury.sol";
+import "../libraries/SafeERC20.sol";
+import "../interfaces/IIncurDebt.sol";
+import "../types/OlympusAccessControlledV2.sol";
+
+contract TokemakIncurDebt is OlympusAccessControlledV2 {
```

Comment:
Let's call this `IncurDebtV1` because it will be used by a couple other DAOs aside from tokemak.

Reply:
Updated